### DISCUSSION TITLE
Toshiba Haori / B10N4KVRG-E Connector Missing

### DISCUSSION THREAD

**Discussion by florianbrede-ayet** [issalig/toshiba_air_cond#38] [Q&A]
Hi,
the "new" Toshiba Haori line does not support wired controllers as per the technical datasheet and does not have a spare cable.
However, I'm pretty sure that the PCB isn't changed from other units, so I gave it a try and disassembled one unit:

![image](https://github.com/issalig/toshiba_air_cond/assets/48515354/eea9162f-dac9-42a1-bf3b-8df5885cca90)

The red wire is coming from the outdoor unit.
Does anyone know if pins at the missing connector (*SERIAL*) are likely to be the TCC link A/B lines?
![image](https://github.com/issalig/toshiba_air_cond/assets/48515354/ce8bffb1-198a-4c53-9b58-448b603f5fca)

**Comment by issalig** [comment]
A/B signal is around 15.6 volts when 1 and 14 when 0. If you have a multimeter check the voltage on those pins and if it is sth around 15 they could be serial.
Also, check the service manual https://bailos.gr/wp-content/uploads/2021/09/20201130_SM_SVM-20079_RAS_R32_HAORI_RAS-B_N4KVRG-E_RAS-_J2AVSG-E1_EN.pdf

Pages 108 and 114 have some info about serial.

**Reply by florianbrede-ayet** [reply]
Seems like I was unlucky with those pads.
I do not measure any voltage between them or against earth.

What I found however, is this terminal:

![image](https://github.com/issalig/toshiba_air_cond/assets/48515354/dfe67760-d521-4462-aa98-91f2083f6460)
![image](https://github.com/issalig/toshiba_air_cond/assets/48515354/31c00021-8296-4854-a440-bf6ba57e8139)

At least on one of those two pins, there is some activity against earth.
Does that look like the right connector or might the latest indoor units have changed?

I'll have to bring my scope and hook it up to see what exactly is going on there.
I also read the manual and checked the functions again and indeed the remote is able to receive error codes from the indoor unit.

Is the photo diode in the remote capable of sending AND receiving signals?
